lib/git-shell-out-strategy.js

```diff
@@ -473,7 +474,10 @@ export default class GitShellOutStrategy {
 
     if (amend) { args.push('--amend'); }
     if (allowEmpty) { args.push('--allow-empty'); }
-    return this.gpgExec(args, {writeOperation: true});
+    const returnValue = await this.gpgExec(args, {writeOperation: true});
```

Comment:
actually, the more I think about it, the more I think we should just log all the git commands in th `exec` function instead of instrumenting individual commands.